br-validations@0.3.0 postinstall: `gulp build`

Problem with gulp build on postinstall

```
npm ERR! Darwin 14.5.0
npm ERR! argv "/Users/aj07mm/.nvm/versions/node/v4.2.1/bin/node" "/Users/aj07mm/.nvm/versions/node/v4.2.1/bin/npm" "install"
npm ERR! node v4.2.1
npm ERR! npm  v2.14.7
npm ERR! code ELIFECYCLE

npm ERR! br-validations@0.3.0 postinstall: `gulp build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the br-validations@0.3.0 postinstall script 'gulp build'.
npm ERR! This is most likely a problem with the br-validations package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     gulp build
npm ERR! You can get their info via:
npm ERR!     npm owner ls br-validations
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/aj07mm/Desktop/orama/minha_conta/orama_web/webapps/webapp/npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -13,8 +13,7 @@
     "url": "https://github.com/the-darc/br-validations/issues"
   },
   "scripts": {
-    "test": "gulp test-coverage",
-    "postinstall": "gulp build"
+    "test": "gulp test-coverage"
   },
   "keywords": [
     "validator",

